### ISSUE TITLE
Process to choose student for the travel money

### ISSUE THREAD

**Issue by dpshelio** [OpenAstronomy/openastronomy#2]
GSoC provides some money to get a student on a conference. The committee needs to agree on a process to choose such student.

**Comment by gradvohl** [comment]
Dear fellows,

  I've been thinking about the process to choose the student for the travel money. I believe that, first, we need to think about the profile of the student we would like to grant. Therefore, I propose that this candidate has the following characteristics, in line with the open development concept that we discussed in our last meeting:

1. Be able to demonstrate that she/he have already made previous contributions to the community and also that she/he is willing to continue making contributions.

2. Demonstrates that she/he belongs to one or more minority groups that we would like to increase participation in the OpenAstronomy community.

3. Be able to collaborate in the event itself, doing a presentation about her/his research or the work she/he is developing.

  In this sense, we would assign a simple score to the items to rank the candidates. I suggest that this score has three possible values: 0 - Ok. 1 - Good. 2 - Great.

  Depending on the number of candidates, we could divide them into equal amounts among the members of the committee. The opinions of the best candidates scored by each committee member are taken to a second round with the entire commission to elect the best candidate.

  To apply, the candidates should send a text of up to 500 words highlighting the 3 items that we will evaluate, as well as the main contributions and what the candidate expects from the event.

   Well, I believe that this proposal is a starting point. Hope this helps. 

  Best regards.

**Comment by bsipocz** [comment]
@gradvohl - This is about GSoC students, so please focus on that rather than a generic selection criteria. 

* they already contributed much more than the avarage applicant to a pyastro conference. They proved that they can collaborate.
* research is irrelevant at pyastro, please do not try to bring back research talks it was bad enough last year
* I'm very much against asking a 500 word essay from the students. they already belong to the community and interacted enough with the mentors that we as mentors can agree on who gets the money. 
* I'm sorry to say, but the decision should be kept within the mentor community and should not be a long process by the openastro committee.

**Comment by gradvohl** [comment]
Sorry about my misunderstanding. But would it not apply to GSoC students, too?

**Comment by bsipocz** [comment]
No, as we said on the call and discussed before, too, there is some money specifically to support one gsoc student. 
That said, the actual participant selection for pyastro is also not the business of the openastro committee.

**Comment by gradvohl** [comment]
Ok.
